generator/lib/behavior/i18n/I18nBehaviorObjectBuilderModifier.php

```diff
@@ -183,7 +183,7 @@ protected function addTranslatedColumnSetter(Column $column)
             $objectBuilder->addMutatorOpenOpen($functionStatement, $column);
         }
         $comment = preg_replace('/^\t/m', '', $comment);
-        $comment = str_replace('@return     ' . $i18nTablePhpName, '@return     ' . $tablePhpName, $comment);
+        $comment = str_replace('@return ' . $i18nTablePhpName, '@return ' . $tablePhpName, $comment);
         $functionStatement = preg_replace('/^\t/m', '', $functionStatement);
```

Comment:
Not fully tested, but maybe replace it with a regular expression that takes any amount of spaces in account:
`$comment = preg_replace('/(@return\s+)(' . preg_quote($i18nTablePhpName, '/') . ')/', '${1}' . preg_replace('/(\$|\\\\)(?=\d)/', '\\\\\1', $tablePhpName), $comment);`